### PR TITLE
feat(superuser): Do not store global selection for non org members

### PR DESCRIPTION
**Pull request by lynnagara**
Do not store any global selection values in local storage for users
who are not org members. This currently only affects superusers.

### CODE REVIEW

File: src/sentry/static/sentry/app/stores/globalSelectionStore.jsx [hunk 1]
```diff
@@ -139,10 +140,12 @@ const GlobalSelectionStore = Reflux.createStore({
   },
 
   updateLocalStorage() {
+    // Do nothing if no org is loaded or user is not an org member
```
Comment (billyvg) [inline src/sentry/static/sentry/app/stores/globalSelectionStore.jsx]: Can we clarify how/why we know that the user is not an org member?

**Review by billyvg — APPROVED** [review]

File: src/sentry/static/sentry/app/stores/globalSelectionStore.jsx (hunk 1 above)
Comment (lynnagara) [reply src/sentry/static/sentry/app/stores/globalSelectionStore.jsx]: Oh, is the assumption on the line below not clear/correct? i.e. not in their organizations store?

Comment (billyvg) [reply src/sentry/static/sentry/app/stores/globalSelectionStore.jsx]: The assumption is correct, I think it'd be good to clarify the reason why organization wouldn't exist in OrganizationStore. e.g. If user is a superuser, `OrganizationsStore` will only contain orgs that they are a member of, while the current organization could be one they can access, but not be a member of.

Comment (lynnagara) [reply src/sentry/static/sentry/app/stores/globalSelectionStore.jsx]: Ok

**Pull request merged**